.eslintrc.json

```diff
@@ -22,10 +25,32 @@
   ],
 
   "rules": {
-    "comma-dangle": ["error", "only-multiline"],
-    "indent": ["error", 2, { "SwitchCase": 1 }],
-    "linebreak-style": ["error", "unix"],
-    "quotes": ["error", "single"],
-    "semi": ["error", "always"]
+    "indent": [
+      2, 
+      4
+    ],
+    "comma-dangle": [
+        "error", 
+        "never"
+    ],
+    "no-unused-vars": [
+        "error", 
+        {
+            "vars": "local", 
+            "args": "none" 
+        }
+    ],
+    "arrow-parens": [
+        2, 
+        "as-needed"
+    ],
+    "no-shadow": "off",
+    "consistent-return": "off",
+    "prefer-promise-reject-errors": "off",
+    "arrow-body-style": "off",
+    "import/no-named-as-default-member": "off",
+    "no-param-reassign": "off",
+    "linebreak-style": "off",
```

Comment:
Don't turn this rule off. Our line breaks should be consistent throughout the project especially since we'll be developing on both mac and windows machines. This will help keep sanity.

If this rule is annoying you should set a default line ending in your editor so it fixes things automatically.